gno.land/pkg/gnoclient/client_txs.go

```diff
@@ -81,6 +98,78 @@ func (c *Client) Call(cfg CallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
 	return c.signAndBroadcastTxCommit(tx, accountNumber, sequenceNumber)
 }
 
+// MultiCall executes a contract call on the blockchain.
+func (c *Client) MultiCall(cfg MultiCallCfg) (*ctypes.ResultBroadcastTxCommit, error) {
```

Comment:
Why not expand the existing `Call` method to support multiple messages?

I think all of our transaction manipulation methods in the client should support 1 or more messages, since they all fit into a single transaction (the result is singular, the payload is not).

Reply:
I just replaced the Call functionality with the Multicall functionality to accomodate your request: [ebaf516](https://github.com/gnolang/gno/pull/1565/commits/ebaf516a36fc8c93705746c7feeea5126e0c439a)